packages/Parallel/Application/ParallelFileProcessor.php

```diff
@@ -185,7 +183,16 @@ function (array $json) use (
                         $this->processPool->quitProcess($processIdentifier);
                         return;
                     }
-
+                    if (!isset($this->processRunCounter[$processIdentifier])) {
+                        $this->processRunCounter[$processIdentifier] = 0;
+                    }
+                    if ($this->processRunCounter[$processIdentifier] >= $this->maxProcessIterations) {
+                        $this->processPool->quitProcess($processIdentifier);
+                        if (is_callable($this->processSpawner)){
```

Comment:
how this can be false at this stage?

Reply:
phpstan complained, I dont think it can be false, but the call structure is complex

Reply:
by following the suggestion in https://github.com/rectorphp/rector-src/pull/3722#discussion_r1187505097 it shouldn't complain, as the local variable will be assigned directly, never null